Makefile

```diff
@@ -435,7 +435,7 @@ istioctl-test: istioctl
 MIXER_TEST_T ?= ${T} ${GOTEST_PARALLEL}
 mixer-test: mixs
 	# Some tests use relative path "testdata", must be run from mixer dir
-	(cd mixer; go test ${GOTEST_P} ${MIXER_TEST_T} ./...)
```

Comment:
So it's about concurrent testing after all?


Does it make more sense to remove ${GOTEST_PARALLEL} in line 435 instead?